sound/soc/sof/sof-audio.h

```diff
@@ -474,6 +477,7 @@ struct snd_sof_pipeline {
 	int started_count;
 	int paused_count;
 	int complete;
+	unsigned int kcps;
```

Comment:
int or s32?

Reply:
the value that's sent out has to be `s32`, because it can be negative for a negative adjustment, and it has to be 32 bits because it's a part of an ABI. But here we're storing a generic integer unsigned value, so I think `unsigned int` is appropriate here.